components/Layout.vue

```diff
@@ -1,52 +1,50 @@
 <template>
-  <div class="relative mx-auto flex max-w-8xl justify-center sm:px-2 lg:px-8 xl:px-12">
-    <div class="hidden lg:relative lg:block lg:flex-none">
-      <div class="absolute inset-y-0 right-0 w-[50vw] bg-slate-50 dark:hidden" />
-      <div class="sticky top-[4.5rem] -ml-0.5 h-[calc(100vh-4.5rem)] overflow-y-auto py-16 pl-0.5">
-        <div class="absolute top-16 bottom-0 right-0 hidden h-12 w-px bg-gradient-to-t from-slate-800 dark:block " />
-        <div class="absolute top-28 bottom-0 right-0 hidden w-px bg-slate-800 dark:block" />
+  <div class="mx-auto max-w-7xl flex flex-col-reverse min-h-full lg:grid lg:grid-cols-5 lg:gap-8 lg:px-8">
+    <div class="hidden lg:relative lg:block lg:flex-none lg:col-span-1">
+      <div class="absolute inset-y-0 right-0 w-[50vw] bg-slate-50" />
+      <div class="sticky top-[4.5rem] h-[calc(100vh-4.5rem)] overflow-y-auto py-16 px-4">
         <Navigation
           :navigation="navigation"
-          class="w-64 pr-8 xl:w-72 xl:pr-16"
+          class="w-full"
         />
       </div>
     </div>
     <div
-      class="min-w-0 max-w-2xl flex-auto px-4 py-16 lg:max-w-none lg:pr-0 lg:pl-8 xl:px-16"
+      class="min-w-full max-w-2xl flex-auto px-4 py-3 lg:py-16 lg:max-w-prose lg:col-span-3"
     >
       <article>
         <header
           :v-if="title || section"
           class="mb-9 space-y-1"
         >
-          <p
+          <h5
```

Comment:
An `h5` tag directly before an `h1` tag isn't great for accessibility. I don't know what the correct tag is here, maybe `span`?

Reply:
Good spot! I've reverted to a `p` tag for now - that should be ok